python/metatensor-torch/tests/atomistic/ase_calculator.py

```diff
@@ -27,162 +22,36 @@
 }
 
 
-class LennardJones(torch.nn.Module):
```

Comment:
Is this intended to be removed?

Reply:
yes! This code now lives in https://github.com/Luthaf/metatensor-lj-test/, which is installed by tox.